test/commands/transaction/create/transfer.test.js

```diff
@@ -101,6 +101,34 @@ describe('transaction:create:transfer', () => {
 			});
 	});
 
+	describe('transaction:create:transfer amount address --data=xxx', () => {
+		setupTest()
+			.command([
+				'transaction:create:transfer',
+				defaultAmount,
+				defaultAddress,
+				'--data=XXXXXX',
+			])
+			.it('should create an tranfer transaction', () => {
+				expect(transactionUtilStub.validateAddress).to.be.calledWithExactly(
+					defaultAddress,
+				);
+				expect(transactionUtilStub.convertLSKToBeddows).to.be.calledWithExactly(
+					defaultAmount,
+				);
+				expect(getInputsFromSources.default).to.be.calledWithExactly({
+					passphrase: {
+						source: undefined,
+						repeatPrompt: true,
+					},
+					secondPassphrase: null,
+				});
+				return expect(printMethodStub).to.be.calledWithExactly(
+					defaultTransaction,
+				);
+			});
+	});
+
```

Comment:
we should add a case for invalid data too

Reply:
as we discussed, we don't need it